js/src/features/project/components/ProjectRoleForm.scss

```diff
@@ -14,6 +22,10 @@
     min-width: 290px;
   }
 
+  &__input {
+    width: 200px !important;
```

Comment:
Is there any other way to write this without `important` ?

Reply:
done

Reply:
done